test/data/dev/dev_data.sql

```diff
@@ -60,10 +60,12 @@ do $report$
         values (i+loop_offset, i+loop_offset);
 
         insert into swrs.fuel (id, report_id, unit_id, fuel_mapping_id, fuel_type, fuel_description, fuel_units, annual_fuel_amount)
-        values (i+loop_offset, i+loop_offset, i+loop_offset, 6, 'Aviation Gasoline (kilolitres)', 'Aviation Gasoline (kilolitres)', 'kl', 1234);
+        values (i+loop_offset, i+loop_offset, i+loop_offset, 96, 'Natural Gas (Sm^3)', 'Natural Gas (Sm^3)', 'Sm^3', 1234);
 
         insert into swrs.emission (id, activity_id, facility_id, fuel_id, naics_id, organisation_id, report_id, unit_id, quantity, calculated_quantity, emission_category, gas_type)
-        values (i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, 1000, 1000, 'BC_ScheduleB_GeneralStationaryCombustionEmissions', 'CO2nonbio');
+        values
+          (i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, 10000, 10000, 'BC_ScheduleB_GeneralStationaryCombustionEmissions', 'CO2nonbio'),
+          (i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, i+loop_offset, 444, 444, 'BC_ScheduleB_FlaringEmissions', 'CH4');
```

Comment:
adding a second emission within this loop means we need to add a sequence to assign ids here, and for all the emission inserts below (inserting `i+loop_offset` into `id` a second time will cause a duplicate id error)

See the `address_sequence` created at the top of the file (line 30-31) and applied below on the address inserts for an example.

Reply:
I went a different route and did some math instead... Not sure if that's better or worse than the sequence actually :thinking:

Reply:
changed to a sequence to be consistent with the `address` table